apps/src/templates/progress/progressHelpers.js

```diff
@@ -181,7 +184,7 @@ export const processedLevel = level => {
     levelNumber: level.kind === LevelKind.unplugged ? undefined : level.title,
     isConceptLevel: level.is_concept_level,
     bonus: level.bonus,
-    pageNumber: level.page_number,
+    pageNumber: level.page_number && level.page_number.toString(),
```

Comment:
I don't think this will work if `page_number` is 0, is that a case we should consider? (not sure if we'd ever end up with a 0 page number..)

Reply:
good catch. i will check for `undefined` to be safe.